application/src/js/cms/_scroll_past_sticky_status_banner.js

```diff
@@ -0,0 +1,36 @@
+function ErrorSummaryWithStickyBanner() {
+}
+
+ErrorSummaryWithStickyBanner.prototype.init = function () {
+    var hrefAnchorRegex = /\#(.+)$/;
+    var errorBox = document.querySelector('.flash-message .alert-box.error');
+
+    if (errorBox != null) {
+        var errorLinks = errorBox.querySelectorAll('a')
+
+        var scrollForStickyHeader = function (event) {
+            /* We use a timeout here so that this code runs after the default click event, which adds an anchor
+             * to the URL and scrolls to the right place. */
+            window.setTimeout(function () {
+                var anchorRegexResult = hrefAnchorRegex.exec(this.getAttribute('href'));
+                if (anchorRegexResult.length === 2) {
+                    var targetElement = document.getElementById(anchorRegexResult[1]);
+
+                    targetElement.scrollIntoView();
+
+                    var statusBanner = document.querySelector('.status-banner.sticky-js');
+                    if (statusBanner != null) {
+                        var statusBannerHeight = statusBanner.clientHeight;
+                        window.scrollBy(0, statusBannerHeight * -1);
```

Comment:
I think `-statusBannerHeight` would be a more descriptive way to do this maths? 😄

Reply:
Sure